ui/src/components/ort-run-info.tsx

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (C) 2024 The ORT Server Authors (See <https://github.com/eclipse-apoapsis/ort-server/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+import { Link, useParams } from '@tanstack/react-router';
+import { Link as Chain } from 'lucide-react';
+
+import {
+  AdvisorJob,
+  AnalyzerJob,
+  EvaluatorJob,
+  ReporterJob,
+  ScannerJob,
+} from '@/api/requests';
+import { Card, CardContent, CardHeader, CardTitle } from '@/components/ui/card';
+import { RunDuration } from './run-duration';
+
+type OrtRunInfoProps = {
+  job?:
+    | AnalyzerJob
+    | AdvisorJob
+    | ScannerJob
+    | EvaluatorJob
+    | ReporterJob
+    | undefined;
+  title: string;
+  icon?: React.ComponentType<{ className?: string }>;
+  nTotal?: number;
+  resultsPath:
+    | 'issues'
+    | 'dependencies'
+    | 'vulnerabilities'
+    | 'rule-violations';
+  pollInterval: number;
+};
+
+export const OrtRunInfo = ({
+  job,
+  title,
+  icon: Icon,
+  nTotal,
+  resultsPath,
+  pollInterval,
+}: OrtRunInfoProps) => {
```

Comment:
I would refactor this component into a more generic StatisticsCard or something that takes in a title, icon, value and maybe something else if required.

Reply:
Maybe later, but I was hoping we could merge this now, without more time-consuming changes, so that I can start working on the groups admin issue.

Reply:
Not just renaming, it shouldn't take in job or anything that specific.